lua/neo-tree/sources/common/components.lua

```diff
@@ -147,4 +147,41 @@ M.name = function(config, node, state)
   }
 end
 
+local prevent_indent = {}
```

Comment:
I didn't notice this before, but this shouldn't be global state. If you want to share some state between components in a render event, put it on the `state` object passed into the function.